Formula/util-linux.rb

```diff
@@ -25,20 +25,21 @@ class UtilLinux < Formula
   keg_only :shadowed_by_macos, "macOS provides the uuid.h header"
 
   depends_on "asciidoctor" => :build
-  depends_on "gettext"
+  depends_on "gettext" => :build
```

Comment:
Might be able to drop entirely on Linux. I think `gettext` is used to convert `po` files, which should already be done for release tarball. On macOS, it is also used for `libintl` which is part of glibc on Linux.
```suggestion
```